src/crypto.rs

```diff
@@ -556,6 +569,58 @@ impl TryFrom<&serde_cbor::Value> for COSEKey {
     }
 }
 
+impl TryFrom<&X509PublicKey> for COSEKey {
+    type Error = WebauthnError;
+    fn try_from(cert: &X509PublicKey) -> Result<COSEKey, Self::Error> {
+        let ec_key = cert
```

Comment:
This should be named public_key not ec_key as it's before we have identified the type.